PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Please use the following structure when proposing changes to our shared Vim configuration and make sure to complete the checklist at the end.
+
+# What
+
+Summarize your changes.
+
+_Example: Specify full path when running focused specs._
+
+# Why
+
+Summarize the motivation for the change (bug, feature, etc.)
+
+_Example: When only a relative path is specified, focused specs don't run if the shell's current working directory differs from the project root._
+
+# Checklist
+
+- [ ] Send RFC email to Braintree developers _if change may be impactful_. Please include a link to this PR so that disucssion about the pros and cons of the change remains linked to the proposed changes. **Recent examples include**: addition of linter and completer, no longer removing end-of-line whitespace on save, changing to fzf for file lookup.
```

Comment:
We should include the email address to send to here.

Reply:
I wanted to make sure I wasn't exposing an alias that wasn't already public. Which one should we use?

Reply:
I don't think we should include the email address.  People who need it know, otherwise I expect it will just get picked up by spam bots.

Reply:
Ah yeah, 👍 .